src/libraries/System.Net.Security/src/System/Net/Security/SslStreamPal.OSX.cs

```diff
@@ -260,9 +260,9 @@ internal static byte[] GetNegotiatedApplicationProtocol(SafeDeleteContext contex
                     }
                 }
 
-                if (inputBuffer.Array != null && inputBuffer.Count > 0)
+                if (inputBuffer != null && inputBuffer.Length > 0)
```

Comment:
The `inputBuffer` span will never be null.  The `inputBuffer != null &&` should be removed.